packages/react/package.json

```diff
@@ -44,6 +44,7 @@
   "dependencies": {
     "@bigmi/client": "workspace:^",
     "@bigmi/core": "workspace:^",
+    "@tanstack/react-query": "^5.74.11",
```

Comment:
This should be a peer dependency because react query relies on shared react context. This can cause multiple versions to be installed and we will have context mismatches, happened to us quite some times 😁